fbw-common/src/systems/instruments/src/EFB/Localization/data/en.json

```diff
@@ -134,6 +134,7 @@
       "EstimatedDuration": "Estimated duration (minutes)",
       "EstimatedTime": "Estimated Time",
       "GSXFuelSyncEnabled": "Call GSX Refuel",
+      "GSXFuelRequested": "GSX Refuel requested",
```

Comment:
Update also needed to localazy keys

Reply:
What is needed here from my Side? ^^

Reply:
This is more a maintainer task, just needs to be added before merge.